### ISSUE TITLE
Check if Ember CLI is running locally (via ember s) in ember-cli-build.js

### ISSUE THREAD

**Issue by Kuzirashi** [ember-cli/ember-cli#5202]
Hey, is it possible to check if Ember CLI is running locally (via ember s) in ember-cli-build.js? Maybe there is some option which could be passed to ember server and then I could read it in ember-cli-build.js? I'm using development build for some deployment outside of localhost and I need to differentiate between locally running ember s and some compiled build (both have the same environment). Custom environment could resolve my problem but I'm unsure about status of it. Here's the issue #1520.

Basically I need to remove xdomain script from index.html when I'm running on localhost because it blocks Ember app from loading.

Thanks in advance.

**Comment by Kuzirashi** [comment]
I'll try luck with accessing some process variables like you do in EmberApp, in ember-cli-build process.env.EMBER_ENV and I'll close if I succeed.

**Comment by stefanpenner** [comment]
> process.env.EMBER_ENV

that is what config/environmennt.js is for.

**Comment by stefanpenner** [comment]
https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/config/environment.js#L3 this file gets executed in the context of node, it builds a pojo, that is shipped to the client and accessible via <app-name>/config/environment

**Comment by Kuzirashi** [comment]
Yeah, thank you. I set ENV.isLocal in config by looking at process.env.LOCAL and then use it in ember-cli-build.js to include/not include xdomain script in index.html. Works fine. :)

**Comment by stefanpenner** [comment]
@Kuzirashi :dancers:
